index.js

```diff
@@ -26,7 +26,15 @@ function lint(textEditor) {
 	const defaultCwd = process.cwd();
 	process.chdir(dir);
 
-	const pkg = loadJson(path.join(dir, 'package.json'));
+	let pkg = loadJson(path.join(dir, 'package.json'));
+
+	// get the parent `package.json` if there's a `"xo": false` in the current one
+	if (pkg.xo !== undefined) {
+		while (pkg.xo === false && dir !== null) {
+			dir = pkgDir.sync(path.join(dir, '..'));
+			pkg = dir === null ? pkg : loadJson(path.join(dir, 'package.json'));
+		}
+	}
 
```

Comment:
That package.json might have XO as a dependency, but decide to use `"xo": false` temporarily or something. I'm just trying to limit the amount of possible issues here. Better safe than sorry.